src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,21 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+};
+
+const booksSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook: (state, action) => {
+      state.books.push(action.payload);
+    },
```

Comment:
- [ ] Great, you can take a look at below snippet to have some insight in order to avoid overwitten of initial book by the new one. It might not be obvious now since we have not worked on component changes.

```
reducers: {
    addBook: (state, action) => {
      state.books = [
        ...state.books,
        {
          id: Math.floor(Math.random() * 100),
          title: action.payload.title,
          author: action.payload.author,
        },
      ]
    }
```